### ISSUE TITLE
MusicSync.sh - Misfire in Error messages

### ISSUE THREAD

**Issue by littlejeem** [littlejeem/sync_scripts#13]
when running, script converts alac and then produces the following

```
rsync completed successfully
alac sync finished
alac not selected
flac processing started
```

the section
```alac not selected```
appears to be wrong/in wrong order

**Comment by littlejeem** [comment]
further log example

```bash
Dec  8 22:39:48  INFO: alac processing started
Dec  8 22:39:48  INFO: files located in /a/b/c/d/

/a/b/c/d/x/y (10 items)
Tagging:
    x - y
URL:
    https://musicbrainz.org/release/a085d0f0-3994-4094-9490-4de3c8736b73
(Similarity: 100.0%) (Digital Media, 2011, XW, [none])
Dec  8 22:40:21 DEBUG: /a/b/c/e/ is empty, no conversion needed
Dec  8 22:40:21  INFO: alac processing finished
Dec  8 22:40:22  INFO: alac sync started
created directory /media/Data_1/Music/Albums
./
x/
x/y/
/x/y/01..m4a
/x/y/09.m4a
/x/y/07.m4a
/x/y/03.m4a
/x/y/06.m4a
/x/y/08.m4a
/x/y/04.m4a
/x/y/05.m4a
/x/y/10.m4a
/x/y/02.m4a
x/y/folder.jpg

sent 225,709,058 bytes  received 375 bytes  26,554,050.94 bytes/sec
total size is 228,091,839  speedup is 1.01
Dec  8 22:40:30  INFO: rsync completed successfully
Dec  8 22:40:30  INFO: alac sync finished
Dec  8 22:40:30  INFO: uploads not selected
Dec  8 22:40:30  INFO: flac processing started
Dec  8 22:40:30  INFO: files located in /a/b/c/d/

/a/b/c/d/x/y (10 items)
Tagging:
    x - y
URL:
    https://musicbrainz.org/release/a085d0f0-3994-4094-9490-3re5c3736b73
(Similarity: 100.0%) (Digital Media, 2011, XW, [none])
Dec  8 22:40:51 DEBUG: /a/b/c/e/ is empty, no conversion needed
Dec  8 22:40:51  INFO: flac processing finished
Dec  8 22:40:52  INFO: flac sync started
created directory /media/Data_1/Music/FLAC_Backups
./

/x/y/z/01.flac
/x/y/z/09.flac
/x/y/z/07.flac
/x/y/z/03.flac
/x/y/z/06.flac
/x/y/z/08.flac
/x/y/z/04.flac
/x/y/z/05.flac
/x/y/z/10.flac
/x/y/z/02.flac
x/y/folder.jpg

sent 212,630,760 bytes  received 381 bytes  22,382,225.37 bytes/sec
total size is 215,626,973  speedup is 1.01
Dec  8 22:41:01  INFO: rsync completed successfully
Dec  8 22:41:01  INFO: flac sync finished
```

**Comment by littlejeem** [comment]
resolved as 47d1ad6f
